catalog/resource_sql_table.go

```diff
@@ -299,7 +299,7 @@ func (ti *SqlTableInfo) diff(oldti *SqlTableInfo) ([]string, error) {
 
 	if ti.TableType == "VIEW" {
 		// View only attributes
-		if ti.ViewDefinition != oldti.ViewDefinition {
+		if strings.TrimSpace(ti.ViewDefinition) != strings.TrimSpace(oldti.ViewDefinition) {
```

Comment:
This shouldn't fix the drift, it will just detect the changes later when the update function is invoked. I've merged #3251 which fixes the configuration drift.

Reply:
I see. Thank you for your comment and fix the view creation issue.